nornir/core/inventory.py

```diff
@@ -105,6 +105,27 @@ def __contains__(self, value: object) -> bool:
         else:
             return any([value == g for g in self])
 
+    def add(self, group: "Group") -> None:
+        """
+        Add the ParentGroup.  The group will only be appended
+        if it does not exist.
+
+        :param group: Parent Group object to add
+        :return: None
+        """
+        # only add the group if it doesn't exist
+        if not self.__contains__(group):
+            self.append(group)
+
+    def remove(self, group: "Group") -> None:
+        """
+        Remove the parent group.
+
+        :param group: Group object to remove
+        :return: None
+        """
+        super().remove(group)
```

Comment:
I think this means the method isn't really necessary as we get the same behaviour without it, right? Or am I missing something?

Reply:
Yeah don't see any reason it needs to stay there.